server/status.js

```diff
@@ -135,7 +135,7 @@ statusEvents.on('connectionActive', (advice) => {
 // Reset online status on startup (users will reconnect)
 const onStartup = (selector) => {
   if (selector == null) {
-    selector = {};
```

Comment:
If you want to keep the same exact behavior is also important to check if `status.idle` is not null or `status.lastActivity` in this selector as the update is also resetting these fields.

Reply:
I see... though i'm not sure what it'd look like. maybe:

`{ $or: [{ "status.online": true }, { "status.idle": { $exists: true } }, { "status.lastActivity": { $exists: true } }] }`

Reply:
Yes, I would like precisely what you did.